app/javascript/packages/verify-flow/steps/password-confirm/password-confirm-step.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import sinon from 'sinon';
+import * as analytics from '@18f/identity-analytics';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { t } from '@18f/identity-i18n';
+import { accordion } from 'identity-style-guide';
+import PasswordConfirmStep from './password-confirm-step';
+
+describe('PasswordConfirm', () => {
```

Comment:
Minor: This should match the component name.

```suggestion
describe('PasswordConfirmStep', () => {
```